test/e2e/opm_test.go

```diff
@@ -45,6 +48,9 @@ var (
 	indexImage1 = "quay.io/olmtest/e2e-index:" + indexTag1
 	indexImage2 = "quay.io/olmtest/e2e-index:" + indexTag2
 	indexImage3 = "quay.io/olmtest/e2e-index:" + indexTag3
+
+	ocpVersion      = "v4.6"
+	redhatOperators = "registry.redhat.io/redhat/redhat-operator-index:" + ocpVersion
```

Comment:
This is a private image -- I think we will run into the issue where we cannot pass in the registry credentials on forks of operator-registry -- meaning this test will only be able to run on merges to master

Reply:
the image is public because it's the one used by all OCP clusters - right?

Reply:
I think you still need auth to access `registry.redhat.io` images -- OCP clusters have these credentials baked in by default somewhere I suppose. 

For example, pulling the index image from a minikube node (with its own docker daemon) results in a 401 unauthorized:
```
docker@minikube:~$ docker pull registry.redhat.io/redhat/redhat-operator-index:v4.6
Error response from daemon: Get https://registry.redhat.io/v2/redhat/redhat-operator-index/manifests/v4.6: unauthorized: Please login to the Red Hat Registry using your Customer Portal credentials. Further instructions can be found here: https://access.redhat.com/RegistryAuthentication
```

Reply:
That being said, checking in registry credentials to github actions and only having this test run on commits to master is still valuable